src/main/java/com/minecolonies/proxy/CommonProxy.java

```diff
@@ -51,6 +51,12 @@ public static NBTTagCompound getEntityData(String name)
     }
 
     @Override
+    public void registerSounds()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1186)